experimental/Origami/src/Origami.jl

```diff
@@ -0,0 +1,312 @@
+# Add your new types, functions, and methods here.
+
+
+module OrigamiHelper
+
+using ..GAP
+
+function __init__()
+    mod_p = "https://ag-weitze-schmithusen.github.io/ModularGroup/PackageInfo.g"
+    ori_p = "https://ag-weitze-schmithusen.github.io/Origami/PackageInfo.g"
+    GAP.Packages.install(mod_p)
+    GAP.Packages.install(ori_p)
+    GAP.Packages.load("Origami")
+end
+
+end
+
+include("types.jl")
+include("canonical.jl")
+include("deck_group.jl")
+include("action.jl")
+include("special_origami.jl")
+include("generate.jl")
+include("homology_action.jl")
+include("cyclic_torus_covers.jl")
+include("homology.jl")
+include("normal_origami.jl")
+include("systoles.jl")
+
+@doc raw"""
+    origami(h::PermGroupElem, v::PermGroupElem)
+
+blablabla
+# Examples
+```jldoctest
+julia> h = @perm (1,2)
+(1,2)
+
+julia> v = @perm (1,2)
+(1,2)
+
+julia> o = origami(h,v)
+Origami ((1,2),(1,2))
```

Comment:
Doctest fails here:
```suggestion
Origami ((1,2),(1,2), 2)
```
But if you just update revised docs and tests that might already resolve it?

Reply:
Yes, my last commit started the documentation and resolves this.